build.sbt

```diff
@@ -42,12 +42,15 @@ lazy val akkahttp = project
   .settings(publishSettings:_*)
   .settings(
     normalizedName := "facebot-akka-http",
-    libraryDependencies += "com.typesafe.akka" %% "akka-http-experimental" % "2.4.11"
+    libraryDependencies ++= Seq(
+      "com.typesafe.akka" %% "akka-http-experimental" % "2.4.11",
+      "org.slf4j" % "slf4j-api" % "1.7.21",
+      "org.slf4j" % "slf4j-simple" % "1.7.21"
```

Comment:
Remove this. Facebot is a library. `slf4j-simple` is a API implementation. This mean an application what depend facebot also will transitively depend `slf4j-simple` and it make impossible to use anther `slf4j-api` implementations.